src/styles/normalizes.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  @include fontMain;
```

Comment:
You don't need the mixin if you use it just once. But ok :)

Reply:
Changed